internal/pkg/cli/svc_deploy.go

```diff
@@ -73,6 +75,7 @@ type deploySvcOpts struct {
 	deployWkldVars
 
 	store               store
+	deployStore *deploy.Store
```

Comment:
Just a nit: seems like the IDE doesn't help you to align the code automatically 🤔